oci-library/oci-hol/java-management/understand-concepts-related-to-management-agent/understand-concepts-related-to-management-agent.md

```diff
@@ -49,7 +49,7 @@ Before the set up of the Management Agent, it is important to understand the con
 
 * **Response File**: For Management Agent installation to take place, a response file is also required. In [Lab 2](?lab=setup-a-fleet), we clicked **Download Install Key** during the creation of our fleet. The file that we downloaded is the response file which contains the install key, as observed in the `ManagementAgentInstallKey` field. The line in our response file `Service.plugin.jms.download=true` will help to download and enable the JMS plugin for Java runtime discovery and reporting.
 
-  The line `Service.plugin.jm.download=true` will also help to download and enable the JMS plugin for Java runtime Lifecycle Management. Lifecycle Management is an advanced feature of JMS, and you may refer to the [Java Lifecycle Management with Java Management Service](../../java-management-lifecycle-management/workshops/freetier/index.html?lab=introduction/introduction) workshop to learn more about it.
+  The line `Service.plugin.jm.download=true` will also help to download and enable the JMS plugin for Java runtime Lifecycle Management. Lifecycle Management is an advanced feature of JMS, and you may refer to the [Perform Java Lifecycle Management with Java Management Service](https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/home) workshop to learn more about it.
```

Comment:
Livelabs reviewer do not allow dynamic links like this. Instead, he propose we put the actual livelabs production link instead

Since we don't have a production URL yet for LCM and OCI services workshop, I'm putting a dummy URL first. Once we are deployed to production, we need to create a new PR with the production URL